### ISSUE TITLE
Navigation Order in Root Expression

### ISSUE THREAD

**Issue by danghoaiphuc** [NSoiffer/MathCAT#315]
Dear @NSoiffer 
We are customizing to have MathCAT read elements in a math expression while editing with a math editor.
We have an issue of navigation order between MathCAT and the editor for root like this: in navigation mode and moving to the right, MathCAT moves to read the content of the root first, then will be the root index like cube or fourth or square root. While, our math editor moves to the root index first, then will move to the root content.
May I know if we could make MathCAT to move to and speak the root index before to the root content when using right arrow?
This will help it speak correctly rather than in reversed wrong value.
Many thanks.

**Comment by NSoiffer** [comment]
Yes, I believe this is possible by writing some rules that are specialized to `mroot` in `vi/navigation.yaml`. Between the holiday this week in the US and some other projects I'm working on, I don't think I'll get to it this week though.

Besides moving left/right, do you also want "zoom in" (down arrow) to land on the root index? This would mean that right moving right would then take you to the contents on the root.
